meta-mel/conf/distro/mel.conf

```diff
@@ -16,12 +16,12 @@ TARGET_VENDOR = "-mel"
 SDK_VENDOR = "-melsdk"
 
 # Distro and release versioning
-DISTRO_VERSION = "12.999"
+DISTRO_VERSION = "13.0+snapshot-${DATE}"
 ARCHIVE_RELEASE_VERSION = "${DISTRO_VERSION}.${BSP_VERSION}.${PATCH_VERSION}"
 PDK_LICENSE_VERSION_DATE = "20181206"
 
 # Version of the mel-scripts artifact, including setup-mel
-SCRIPTS_VERSION ?= "0"
+SCRIPTS_VERSION ?= "1"
```

Comment:
do we really want to bump this here? there have been no product releases with kirkstone/master